easybuild/easyblocks/g/gromacs.py

```diff
@@ -182,6 +182,12 @@ def sanity_check_step(self):
         if self.toolchain.options.get('usempi', None):
             suff = '_mpi'
 
+        # Add the _d suffix to the suffix, in case of the double precission
+        if '-DGMX_DOUBLE=on' in self.cfg['configopts']:
+            suff = suff + '_d'
+        elif '-DGMX_DOUBLE=ON' in self.cfg['configopts']:
+            suff = suff + '_d'
```

Comment:
@paderijk this is exactly what I was 'afraid' of, lot's of copy-pasting and checking for something that's actually very trivial, there are probably other ways of setting `GMX_DOUBLE` such that double precision is enabled...

From the CMake documentation: "True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number".

@besserox since this is CMake basically, thoughts?